patterns/grok-patterns

```diff
@@ -1,6 +1,6 @@
 USERNAME [a-zA-Z0-9._-]+
 USER %{USERNAME}
-EMAILLOCALPART [a-zA-Z0-9!#$%&'*+\-/=?^_`{|}~]+(?:\.[a-zA-Z0-9!#$%&'*+\-/=?^_`{|}~]+)*
+EMAILLOCALPART [a-zA-Z0-9!#$%&'*+\-/=?^_`{|}~]{1,64}(?:\.[a-zA-Z0-9!#$%&'*+\-/=?^_`{|}~]{1,62}){0,63}
```

Comment:
@yaauie incorporated your suggestion and added these "tweaks" on top ([local-part is limited to 64 characters](https://tools.ietf.org/html/rfc5321#section-4.5.3.1.1)).
we can not validate the dots '.' version to be < 64 chars but could at least help the engine have one less pattern to potentially blow up looping "forever".

Reply:
realized this might be considered a regression to be in a non-major release, going to revert and only handle the 64 limit in ECS mode.

Reply:
or to save some cycles simply re-target the PR to only affect the new ECS pattern set (keeping legacy as is).